### ISSUE TITLE
Is it possible to bind {...props} in Vue? $attrs is undefined

### ISSUE THREAD

**Issue by pakerwreah** [GeekyAnts/vue-native-core#106]
I'm trying hard to create a custom component in vue-native but I can't propagate props to children.

**Comment by neeraj-singh47** [comment]
@pakerwreah Did you try passing it as normal props?

**Comment by pakerwreah** [comment]
Yes I did, it works, but I like to make generic components so my developer team can work seamless. As it is I'll have to map every single prop to my child component. In normal Vue I would just v-bind='$attrs' but $attrs is not even defined in vue native and I tried binding a json manually just for testing v-bind="{ placeholder: 'test 123' }" and it doesn't work either.
